webapp/channels/src/components/search_results/search_results.tsx

```diff
@@ -191,19 +190,19 @@ const SearchResults: React.FC<Props> = (props: Props): JSX.Element => {
         titleDescriptor.defaultMessage = 'Recent Mentions';
     } else if (isFlaggedPosts) {
         noResultsProps.variant = NoResultsVariant.FlaggedPosts;
-        noResultsProps.subtitleValues = {icon: <FlagIcon className='icon  no-results__mini_icon'/>};
+        noResultsProps.subtitleValues = {buttonText: <strong>{Utils.localizeMessage('flag_post.flag', 'Save Message')}</strong>};
```

Comment:
Instead of using `Utils.localizeMessage` here, could you change these to use `useIntl` and `intl.formatMessage`?

I would've suggested to change this to use rich formatting to only need a single translation string for this text, but I think it makes sense to split them so that this will always use the same text as the button even if that changes